tests/CoreCLR/runtest/runtest.cmd

```diff
@@ -104,6 +104,11 @@ if %__VSToolsRoot:~-1%==\ set "__VSToolsRoot=%__VSToolsRoot:~0,-1%"
 
 set _msbuildexe="%VSINSTALLDIR%\MSBuild\15.0\Bin\MSBuild.exe"
 
+if not exist !_msbuildexe! (
```

Comment:
Nit: could also just make this decision part of the `if defined VS160COMNTOOLS (` detection at the beginning of the script (the fact that this is a Dev15/16 difference becomes self-documenting)